### DISCUSSION TITLE
Saleor Dashboard is not responsive on Mobile View

### DISCUSSION THREAD

**Discussion by herobaby71** [mirumee/saleor-dashboard#3563] [General]
Saleor Dashboard is not responsive on Mobile View. Hi guys, do you know if Saleor Dashboard design will be modified to support Mobile devices? Currently, the user experience is not the best across mobile devices.

![image](https://user-images.githubusercontent.com/26393308/234173756-298f1e08-5411-41e1-b897-d3d3912f67ed.png)

**Comment by timuric** [comment]
hey @herobaby71 we are making improvements to responsive layouts, feel free to open PR if it is a blocker for you
